tools/check_format_test_helper.py

```diff
@@ -5,6 +5,7 @@
 # docker. Normally this is run via check_format_test.sh, which
 # executes it in under docker.
 
+from __future__ import print_function
```

Comment:
Can you be consistent in putting a separating blank line between the `__future__` imports and the rest? I think the space makes sense, but it's not done everywhere.

Reply:
Fixed.